packages/@vuepress/theme-default/src/client/components/ToggleDarkButton.vue

```diff
@@ -49,12 +49,29 @@ const setDarkClass = (): void => {
   htmlEl?.classList.toggle('dark', isDark.value)
 }
 
+const updateThemeColor = (): void => {
+  const themeColorEl = window?.document.querySelectorAll('meta[name="theme-color"]')
+  const newThemeColor = window.getComputedStyle(
```

Comment:
Consistently use `window?.`

Or skip the following code when `themeColorEl` is empty

Reply:
https://github.com/vuepress/vuepress-next/blob/b84e5dbdf766fe41834d215696f0406647b7cd20/packages/%40vuepress/theme-default/src/client/components/ToggleDarkButton.vue#L52-L67

Thanks for reviewing again, I prefer skip the following code when `themeColorEl` or `newThemeColor` is empty (not very sure the additional check for `newThemeColor` is necessary)